### ISSUE TITLE
Interactive book throwing error

### ISSUE THREAD

**Issue by Omegaviv** [CircuitVerse/mobile-app#216]
**Describe the bug**
In some chapters of interactive book, when moving to the next page ```_inlines.isEmpty': is not true.``` error is thrown as shown in the video below

**To Reproduce**

1. Run the app
2. Go to Interactive books
3. Go to the sequential SSI section 
4. Open the flip-flops section
5. Keep going to the next pages and see the error

**Console log**
```
======== Exception caught by widgets library =======================================================
The following assertion was thrown building RawGestureDetector-[LabeledGlobalKey<RawGestureDetectorState>#f9896](state: RawGestureDetectorState#a0167(gestures: <none>, behavior: opaque)):
'package:flutter_markdown/src/builder.dart': Failed assertion: line 180 pos 12: '_inlines.isEmpty': is not true.

The relevant error-causing widget was: 
  SingleChildScrollView SingleChildScrollView:file:///D:/open_source/CircuitVerse/mobile-app/lib/ui/views/ib/ib_page_view.dart:531:22
When the exception was thrown, this was the stack: 
#2      MarkdownBuilder.build (package:flutter_markdown/src/builder.dart:180:12)
#3      _MarkdownWidgetState._parseMarkdown (package:flutter_markdown/src/widget.dart:336:25)
#4      _MarkdownWidgetState.didChangeDependencies (package:flutter_markdown/src/widget.dart:280:5)
#5      StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:4749:11)
#6      ComponentElement.mount (package:flutter/src/widgets/framework.dart:4561:5)
#7      Element.inflateWidget (package:flutter/src/widgets/framework.dart:3631:14)
#8      MultiChildRenderObjectElement.inflateWidget (package:flutter/src/widgets/framework.dart:6261:36)
#9      Element.updateChild (package:flutter/src/widgets/framework.dart:3383:18)
#10     RenderObjectElement.updateChildren (package:flutter/src/widgets/framework.dart:5684:32)
#11     MultiChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6284:17)
#12     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#13     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6130:14)
#14     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#15     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6130:14)
#16     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#17     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6130:14)
#18     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#19     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6130:14)
#20     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#21     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6130:14)
#22     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#23     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6130:14)
#24     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#25     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4613:16)
#26     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:4763:11)
#27     Element.rebuild (package:flutter/src/widgets/framework.dart:4311:5)
#28     StatefulElement.update (package:flutter/src/widgets/framework.dart:4795:5)
#29     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#30     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6130:14)
#31     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#32     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4613:16)
#33     Element.rebuild (package:flutter/src/widgets/framework.dart:4311:5)
#34     ProxyElement.update (package:flutter/src/widgets/framework.dart:4943:5)
#35     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#36     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6130:14)
#37     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#38     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4613:16)
#39     Element.rebuild (package:flutter/src/widgets/framework.dart:4311:5)
#40     StatelessElement.update (package:flutter/src/widgets/framework.dart:4669:5)
#41     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#42     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6130:14)
#43     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#44     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6130:14)
#45     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#46     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6130:14)
#47     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#48     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4613:16)
#49     Element.rebuild (package:flutter/src/widgets/framework.dart:4311:5)
#50     StatelessElement.update (package:flutter/src/widgets/framework.dart:4669:5)
#51     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#52     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4613:16)
#53     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:4763:11)
#54     Element.rebuild (package:flutter/src/widgets/framework.dart:4311:5)
#55     StatefulElement.update (package:flutter/src/widgets/framework.dart:4795:5)
#56     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#57     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4613:16)
#58     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:4763:11)
#59     Element.rebuild (package:flutter/src/widgets/framework.dart:4311:5)
#60     StatefulElement.update (package:flutter/src/widgets/framework.dart:4795:5)
#61     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#62     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4613:16)
#63     Element.rebuild (package:flutter/src/widgets/framework.dart:4311:5)
#64     StatelessElement.update (package:flutter/src/widgets/framework.dart:4669:5)
#65     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#66     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6130:14)
#67     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#68     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6130:14)
#69     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#70     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6130:14)
#71     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#72     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4613:16)
#73     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:4763:11)
#74     Element.rebuild (package:flutter/src/widgets/framework.dart:4311:5)
#75     StatefulElement.update (package:flutter/src/widgets/framework.dart:4795:5)
#76     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#77     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6130:14)
#78     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#79     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6130:14)
#80     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#81     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4613:16)
#82     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:4763:11)
#83     Element.rebuild (package:flutter/src/widgets/framework.dart:4311:5)
#84     StatefulElement.update (package:flutter/src/widgets/framework.dart:4795:5)
#85     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#86     SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6130:14)
#87     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#88     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4613:16)
#89     Element.rebuild (package:flutter/src/widgets/framework.dart:4311:5)
#90     StatelessElement.update (package:flutter/src/widgets/framework.dart:4669:5)
#91     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#92     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4613:16)
#93     Element.rebuild (package:flutter/src/widgets/framework.dart:4311:5)
#94     StatelessElement.update (package:flutter/src/widgets/framework.dart:4669:5)
#95     Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#96     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4613:16)
#97     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:4763:11)
#98     Element.rebuild (package:flutter/src/widgets/framework.dart:4311:5)
#99     StatefulElement.update (package:flutter/src/widgets/framework.dart:4795:5)
#100    Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#101    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4613:16)
#102    Element.rebuild (package:flutter/src/widgets/framework.dart:4311:5)
#103    StatelessElement.update (package:flutter/src/widgets/framework.dart:4669:5)
#104    Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#105    RenderObjectElement.updateChildren (package:flutter/src/widgets/framework.dart:5626:32)
#106    MultiChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6284:17)
#107    Element.updateChild (package:flutter/src/widgets/framework.dart:3370:15)
#108    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4613:16)
#109    Element.rebuild (package:flutter/src/widgets/framework.dart:4311:5)
#110    BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2578:33)
#111    WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:882:21)
#112    RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:363:5)
#113    SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:1145:15)
#114    SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:1082:9)
#115    SchedulerBinding._handleDrawFrame (package:flutter/src/scheduler/binding.dart:996:5)
#119    _invoke (dart:ui/hooks.dart:150:10)
#120    PlatformDispatcher._drawFrame (dart:ui/platform_dispatcher.dart:270:5)
#121    _drawFrame (dart:ui/hooks.dart:114:31)
(elided 5 frames from class _AssertionError and dart:async)
====================================================================================================
```

**Video**

https://user-images.githubusercontent.com/86337946/159200099-7d903421-ffac-460c-a25e-3c8021cc590b.mp4

**Are you working on this issue? (Yes/No)**
Yes

**Comment by Omegaviv** [comment]
Any suggestions on how this could be solved?

**Comment by Omegaviv** [comment]
<img src="https://user-images.githubusercontent.com/86337946/159844751-aa25a19b-ba91-4428-87e0-62ca79b77327.jpg" width="200">

According to me the content for Cock Signals are missing or some error int the data base

**Comment by Tanmoy741127** [comment]
@tachyons I have able to reproduced the error,  can you assign this to me to fix ?

**Comment by Tanmoy741127** [comment]
OK 👍

**Comment by Tanmoy741127** [comment]
@tachyons 
I have fixed the bug and created PR at #249 
Please check

**Comment by Tanmoy741127** [comment]
@tachyons the issue has been fixed. So you can close this issue
